anycable-go/values.yaml

```diff
@@ -104,6 +104,12 @@ env:
   # Redis channel for broadcasts
   anycableRedisChannel: "__anycable__"
 
+  # Comma separated list of sentinel hosts, format: ':password@hostname:port,..'
+  anycableRedisSentinels: ""
+
+  # Interval to rediscover sentinels in seconds, default: 30
+  anycableRedisSentinelDiscoveryInterval: ""
```

Comment:
Set 30 explicitly, I guess.

Reply:
That will override anycable-go's default. I'm not sure that it is good

Reply:
Fixed